core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1356,6 +1367,35 @@
 
   public static final String CARBON_UPDATE_SYNC_FOLDER_DEFAULT = "/tmp/carbondata";
 
+  /**
+   * S3 Constants
+   */
+
+  public static final String S3_ACCESS_KEY = "fs.s3a.access.key";
```

Comment:
i think its better not to keep the s3 specific property is CarbonCommonConstant
fs.s3a.access.key, fs.s3a.secret.key, fs.s3a.impl
these constant already would have been defined in s3 .